src/benchmark/Akka.Benchmarks/Serialization/SerializationBenchmarks.cs

```diff
@@ -47,6 +56,16 @@ public void Json_serializer_message_roundtrip()
             }
         }
 
+        [Benchmark(OperationsPerInvoke = Operations)]
+        public void Json_serializer_large_message_roundtrip()
+        {
+            for (int i = 0; i < Operations; i++)
+            {
+                var bytes = json.ToBinary(message);
```

Comment:
I totally missed this copypasta. Working on fix and will include fresh runs of benchmarks.